Add new recycle bin options for Security and Workflow events

remiremi opened the pull request:
This pull requests adds a "Deleted XXX" option to the following okta pages

* security > authenticator
* security > authentication policies
* security > global session policies
* security > profile enrollment policies
* security > identity providers
* security > network zone
* security > apis
* workflow > inline hooks
* workflow > event hooks

Clicking "Deleted XXX" will open the usual window with a table listing deleted items, with the ability to select items and click "restore with backupta". This will take the user to the "events" page of backupta, with the items pre-filtered. 

The events from "authenticator", "inline hooks", and "event hooks" will show up in backupta however they are not revertable. All other events are revertable.

gabrielsroka commented:
i see code for OIE, but not for Classic. do those events not happen? or is Backupta unable to restore those objects?

remiremi commented:
@gabrielsroka I updated the pull request and added recycle bins for both "Security > Authentication" and "Security > Multifactor" for Classic tenants.

gabrielsroka commented:
looks good. i delete the trailing commas and added missing semicolons.

i also changed the OIE ones to have "OIE" -- mirroring the Classic ones.

i'll merge the change here and submit the updated extensions to Google
